gatsby-node.js

```diff
@@ -163,24 +169,22 @@ exports.sourceNodes = async ({
     }),
     // TODO: Switch this out for real Referendum data once edge cases get fixed
     // Referendums: election.Referendums.map(referendum => {
-    Referendums: DUMMY_DATA.elections.Elections["11/3/2020"].Referendums.map(
-      referendum => {
-        const id = createNodeId(`${REFERENDUM_NODE_TYPE}-${election.Date}`)
+    Referendums: election.Referendums.map(referendum => {
+      const id = createNodeId(`${REFERENDUM_NODE_TYPE}-${election.Name}`)
```

Comment:
I think this should use the referendum name instead of the election name - otherwise they'll have the same id

Reply:
Fixed!